index.html

```diff
@@ -584,7 +604,8 @@ <h1>How can I make a button go full width?</h1>
       </div>
     </footer>
 
-    <div id="modal-card" class="modal modal-fx-3dSlit">
+    <!-- TODO: aria-modal should be set to true when modal is opened -->
+    <div id="modal-card" class="modal modal-fx-3dSlit" role="dialog" aria-modal="false">
```

Comment:
Also as part of improvement you can `aria-describedby` and `aria-labelledby` attr
Please check example here https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-modal